### ISSUE TITLE
So birthday...

### ISSUE THREAD

**Issue by IlywI** [Backdash/MonikaModDev#1050]
So on your birthday maybe get her (in game) to tell you to go check the game directory(like what she did for surprise.txt) and she makes a file called happybirthday.txt and within the txt, just write stuff to wish <y/n> on their birthday. Like maybe some very heartwarming stuff? I dont english :V so i can't write much.

**Comment by 8lack1ce** [comment]
yeah a birthday thing would be nice.  I'd be a little bummed if my supreme waifu goddess didn't acknowledge my birthday

**Comment by ACoolCat** [comment]
It'd be cool if she had a sprite swap for it, like for the other holidays. Maybe a birthday hat with some balloons and streamers. Maybe she even gets a present for you.

**Comment by Ulvsblakk** [comment]
As cool as that would be, I don't know if that'd be easy. Just because everyone has a different birthday (for the most part). You'd have to make a separate event where she asks for your birthday, and then a whole new one just for the festivities.

**Comment by YayPepsi** [comment]
Maybe the party hat and stuff can be something you add yourself and it stays for one day? Like the rose object.  But I'm not really sure how the rose object worked, so I don't know if that would be something that could actually be done.

**Comment by 8lack1ce** [comment]
She could ask when your birthday is, then a calendar menu pops up, you pick your birthday, and then when the day comes, she celebrates
